objects/rct2/ride/rct2.kart1.json

```diff
@@ -84,8 +84,8 @@
             "ru-RU": "Картинг"
         },
         "description": {
-            "en-GB": "Self-drive petrol-engined go karts",
-            "en-US": "Self-drive gas-engined go karts",
+            "en-GB": "Riders race each other in self-drive petrol-engined go-karts",
```

Comment:
I prefer reverting the description change (apart from the spelling of "go-karts"). This description is only shown in the vehicle window, for which I think the old description is more appropriate.